docs/examples/loaders/OBJLoader2.html

```diff
@@ -52,7 +52,7 @@ <h3>[name]( [param:LoadingManager manager], [param:LoaderSupport.ConsoleLogger l
 
 		<h2>Methods</h2>
 
-		<h3>[method:Object3D parse]( {[param:arraybuffer content]|[param:String content]] )</h3>
+		<h3>[method:Object3D parse]( [param:arraybuffer content]|[param:String content]] )</h3>
```

Comment:
it should be

```suggestion
		<h3>[method:Object3D parse]( [param:arraybuffer content]|[param:String content] )</h3>
```

Reply:
@bakerman Can you please commit the suggestion by @michaelsauget?